README.rst

```diff
@@ -81,6 +81,7 @@ Also, as a convenience, these names from the ``mock`` module are accessible dire
 * `call <https://docs.python.org/3/library/unittest.mock.html#call>`_ *(Version 1.1)*
 * `sentinel <https://docs.python.org/3/library/unittest.mock.html#sentinel>`_ *(Version 1.2)*
 * `mock_open <https://docs.python.org/3/library/unittest.mock.html#mock-open>`_
+* `seal <https://docs.python.org/3/library/unittest.mock.html#unittest.mock.seal>`_ *(Version ???)*
```

Comment:
Wasn't sure what to put for the version string here. Semver would imply that the next version would be 3.4, but I don't know if there's a major release coming up.

Reply:
You are correct, the next version would be 3.4 given the new feature, please update it. 👍